docs/api-reference/enrichment-components/upgrade-guides/6-0-x-upgrade-guide/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "6.0.x Upgrade Guide"
+sidebar_position: 50
+---
+
+In version 6.0.0, Enrich is refactored to use [common-streams](https://github.com/snowplow-incubator/common-streams) libraries under the hood. [common-streams](https://github.com/snowplow-incubator/common-streams) is the collection of libraries that contains streaming-related constructs commonly used across many Snowplow streaming applications. 
+
+[common-streams](https://github.com/snowplow-incubator/common-streams) allows for the adjustment of many different settings. It also provides default values for most of these settings, which are battle-tested. Therefore, we recommend using default values whenever it is possible. You can find more information about defaults in [configuration reference](/docs/api-reference/enrichment-components/configuration-reference/index.md).
+
+Also, we took this opportunity to make a few breaking changes. Here are the changes:
+
+### Config Field Changes
+
+In version 6.0.0, some of the config fields are renamed or moved to a different section. Here are these changes:
+
+* `incomplete` stream is renamed to `failed`.
+
+* `acceptInvalid` and `exitOnJsCompileError` fields under `featureFlags` section are moved under the `validation` section.
+
+* `experimental.metadata` section is moved to the root level.
+
+* In enrich-kafka, `output.good.headers` field is renamed to `output.good.attributes`.
+
+* In enrich-kafka, `blobStorage.azureStorage.accounts` section is moved to the `blobClients.accounts`.
+
+### Feature Deprecations
+
+* Output `pii` stream is removed. There will no longer be an option to write `pii_transformation` events to an extra output stream.
+
+* Remote adapters are removed.
```

Comment:
```suggestion
* Remote adapters are removed. This was another feature with little to no usage that allowed Enrich to support custom payloads (which would be sent to a configured URL for translation into the expected format). In practice, most of this can be already achieved with [Iglu Webhooks](/docs/sources/webhooks/iglu-webhook/index.md).
```